### ISSUE TITLE
Errors in DX11 Shader (NVIDIA 1070)

### ISSUE THREAD

**Issue by thePostFuturist** [unitycoder/UnityPointCloudViewer#37]
```
'UnityObjectToClipPos': cannot implicitly convert from 'float4x4' to 'float4'
Compiling Vertex program
Platform defines: UNITY_NO_DXT5nm UNITY_ENABLE_REFLECTION_BUFFERS UNITY_NO_CUBEMAP_ARRAY UNITY_USE_DITHER_MASK_FOR_ALPHABLENDED_SHADOWS UNITY_PBS_USE_BRDF1 UNITY_SPECCUBE_BOX_PROJECTION UNITY_SPECCUBE_BLENDING UNITY_ENABLE_DETAIL_NORMALMAP SHADER_API_DESKTOP UNITY_COLORSPACE_GAMMA UNITY_LIGHT_PROBE_PROXY_VOLUME
```

**Comment by unitycoder** [comment]
this should be fixed in previous version already
https://github.com/unitycoder/UnityPointCloudViewer/issues/28

which plugin version you are using and which unity version?

**Comment by unitycoder** [comment]
oh i see its different issue, see this one
https://forum.unity.com/threads/released-point-cloud-viewer-tools.240536/page-6#post-3296465

**Comment by unitycoder** [comment]
fixed in v1.92
